lib/Service/TableTemplateService.php

```diff
@@ -741,45 +733,59 @@ private function makeStartupTable(Table $table):void {
 			$columns['ease']->getId() => 5,
 			$columns['done']->getId() => 'false',
 		]);
 		$this->createRow($table, [
-			// TRANSLATORS This is an example name
 			$columns['what']->getId() => $this->l->t('Edit a row'),
-			// TRANSLATORS This is an example account manager
-			$columns['how']->getId() => 'Hover the mouse over a row you want to edit. Click on the pen on the right side. Maybe you want to add a "done" status to this row.',
+			// TRANSLATORS: the asterisks (*) are used to format the icon name and column name in italics
+			$columns['how']->getId() => $this->l->t('Go to a row you want to edit and use the *pencil* edit button. Maybe you want to add a *Done* status to this row?'),
 			$columns['ease']->getId() => 5,
 			$columns['done']->getId() => 'false',
 		]);
 		$this->createRow($table, [
 			// TRANSLATORS This is an example name
 			$columns['what']->getId() => $this->l->t('Add a new column'),
-			// TRANSLATORS This is an example account manager
-			$columns['how']->getId() => 'You can add, remove and adjust columns as you need it. Click on the three-dot-menu on the upper right of this table and choose "create column". Fill in the data you want, at least a title and column type.',
+			// TRANSLATORS: the asterisks (*) are used to format the menu entry title in italics
+			$columns['how']->getId() => $this->l->t('You can add, remove and adjust columns as you need. Open the three-dot-menu on the upper right of this table and choose *Create column*. Fill in the data you want, at least a title and column type.'),
 			$columns['ease']->getId() => 4,
 			$columns['done']->getId() => 'false',
 		]);
 		$this->createRow($table, [
-			// TRANSLATORS This is an example name
+			$columns['what']->getId() => $this->l->t('Create views for tables'),
+			$columns['how']->getId() => $this->l->t('Filter data and save table presets as views to share and combine them into applications.'),
+			$columns['ease']->getId() => 4,
+			$columns['done']->getId() => 'false',
+		]);
+		$this->createRow($table, [
+			$columns['what']->getId() => $this->l->t('Create applications'),
+			$columns['how']->getId() => $this->l->t('Combine different tables and views into no-code applications for any purpose. This makes them easily accessible directly in the app bar.'),
+			$columns['ease']->getId() => 3,
+			$columns['done']->getId() => 'false',
+		]);
+		$this->createRow($table, [
 			$columns['what']->getId() => $this->l->t('Read the docs'),
-			// TRANSLATORS This is an example account manager
-			$columns['how']->getId() => 'If you want to go through the documentation, this can be found here: https://github.com/nextcloud/tables/wiki',
+			// TRANSLATORS: the link is formatted using markdown, the pattern is: [Title](URL)
+			$columns['how']->getId() => $this->l->t('If you want to go through the documentation, this can be found here: [Nextcloud Tables documentation](https://github.com/nextcloud/tables/wiki)'),
```

Comment:
oh, i guess the URL should be submitted as a parameter :facepalm: tomorrow then, shutting down for today.